instances/latest/commonCoordinateSpaceVersions/MBISC/MBISC_v2012.jsonld

```diff
@@ -0,0 +1,125 @@
+{
+    "@context": {
+        "@vocab": "https://openminds.om-i.org/props/"
+    },
+    "@id": "https://openminds.om-i.org/instances/commonCoordinateSpaceVersion/MBISC_v2012",
+    "@type": "https://openminds.om-i.org/types/CommonCoordinateSpaceVersion",
+    "abbreviation": "MBISC_v2012",
+    "author": null,
+    "custodian": null,
+    "description": "'The Marmoset Brain in Stereotaxic Coordinates' (2012), page IX, Figure B and \"Histology\" section explains: The horizontal zero plane passes through the lower margin of the orbit and the center of the external auditory meatus. The anteroposterior zero plane is perpendicular to the horizontal zero plane and passes through the centers of the external auditory meatus. The left-right zero plane is the midsagittal plane.",
+    "digitalIdentifier": {
+        "@id": "https://openminds.om-i.org/instances/ISBN/978-0-12-415818-4"
+    },
+    "fullName": "The Marmoset Brain in Stereotaxic Coordinates",
+    "homepage": "http://www.neura.edu.au/research/themes/paxinos-group",
+    "howToCite": "G. Paxinos, C. Watson, M. Petrides, M. Rosa, H. Tokuno, The Marmoset Brain in Stereotaxic Coordinates, Academic Press (2012).",
+    "ontologyIdentifier": null,
+    "shortName": "MBISC",
+    "accessibility": {
+        "@id": "https://openminds.om-i.org/instances/productAccessibility/freeAccess"
+    },
+    "anatomicalAxesOrientation": {
+        "@id": "https://openminds.om-i.org/instances/anatomicalAxesOrientation/RAS"
+    },
+    "versionIdentifier": "v2012",
+    "fullDocumentation": {
+        "@type": "https://openminds.om-i.org/types/WebResource",
+        "contentDescription": null,
+        "format": "https://openminds.om-i.org/instances/contentTypes/application_pdf",
+        "IRI": "https://r.marmosetbrain.org/Atlas+Small.pdf"
+    },
```

Comment:
```suggestion
    "fullDocumentation": {
        "@id": "https://r.marmosetbrain.org/Atlas+Small.pdf"
    },
```

cannot be a nested object but must be linked. For PIDs and IRIs we use the resolvable PID or the IRI path as atid. It is then up to adopting systems to match/resolve them to corresponding entries (EBRAINS is doing this)